account_payment_select_account/views/account_payment_view.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" ?>
+<!--
+    Copyright (C) 2016-Today: La Louve (<http://www.cooplalouve.fr/>)
+    Copyright (C) 2019-Today: Druidoo (<https://www.druidoo.io>)
+    @author: La Louve
+    License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html
+-->
+<odoo>
+    <record id="view_account_payment_form" model="ir.ui.view">
+        <field name="model">account.payment</field>
+        <field name="inherit_id" ref="account.view_account_payment_form" />
+        <field name="arch" type="xml">
+            <field name="journal_id" position="attributes">
+                <attribute
+                    name="options"
+                >{'no_create': True, 'no_quick_create': True, 'no_create_edit': True}</attribute>
+            </field>
+
+            <field name="journal_id" position="after">
+                <field name="require_payment_account" invisible="1" />
+                <field
+                    name="payment_account_id"
```

Comment:
When clicking on `Create Payment`, if `require_payment_account` is True and `payment_account_id` is set, update account_payment.outstanding_account_id = `payment_account_id`
<img width="975" height="299" alt="image" src="https://github.com/user-attachments/assets/9b0bced5-0e68-4d93-9fda-cf02c4af9697" />